config/fxdata/terrain.cfg

```diff
@@ -10,187 +10,835 @@ RoomsCount = 17
 [slab0]
 Name = HARD
 TooltipTextID = 324
+BlockFlagsHeight = 4
```

Comment:
Could you add some comments here, describing what the options do where it's know. And indicate which values are possible.

Reply:
Just in slab0?

Reply:
That's how the other configs did it. Comment the properties on the first item in the list.